client/src/main/java/org/apache/cassandra/sidecar/client/HttpResponseImpl.java

```diff
@@ -32,17 +32,22 @@ public class HttpResponseImpl implements HttpResponse
     private final String statusMessage;
     private final byte[] raw;
     private final Map<String, List<String>> headers;
+    private final SidecarInstance respondingServer;
```

Comment:
NIT: instance for consistency with the `SidecarInstance` class name
```suggestion
    private final SidecarInstance sidecarInstance;
```